packages/trace-viewer/src/ui/workbenchLoader.tsx

```diff
@@ -24,7 +24,8 @@ import { Workbench } from './workbench';
 import { TestServerConnection } from '@testIsomorphic/testServerConnection';
 
 export const WorkbenchLoader: React.FunctionComponent<{
-}> = () => {
+  embedded?: boolean
```

Comment:
```suggestion
  supportTraceDrop?: boolean
```

Reply:
`embedded` mode only hides the trace viewer header, so maybe call it `hideHeader` instead?